docs/user/ml/index.asciidoc

```diff
@@ -142,3 +142,27 @@ deviation and rerun the analysis with the modified values.
 [role="screenshot"]
 image::user/ml/images/ml-explain-log-rate.png[Log rate spike explained]
 
+
+[discrete]
+[[log-pattern-analysis]]
+=== Log pattern analysis
+
+preview::[]
+
+Log pattern analysis helps you to find patterns in unstructured log messages and 
+makes it easier to examine your data. It performs categorization analysis on a 
+selected field of a {data-source}, creates categories based on the data and 
+displays them together with a chart that shows the distribution of each category 
+and an example document that matches the category.
+
+You can find log pattern analysis under **{ml-app}** > **AIOps Labs** where you 
+can select the {data-source} or saved search that you want to analyze.
+
+[role="screenshot"]
+image::user/ml/images/ml-log-pattern-analysis.png[Log pattern analysis UI]
```

Comment:
> image::user/ml/images/ml-log-pattern-analysis.png[Log pattern analysis UI]

Though not strictly part of this PR, I was surprised to see each word capitalized in "Log Pattern Analysis". It doesn't align with "Explain log rate spikes" in the earlier screenshot and doesn't align with the general advice in https://elastic.github.io/eui/#/guidelines/writing/guidelines#capitalization. Might be nice to change that to sentence case then re-take the screenshots.

Reply:
https://github.com/elastic/kibana/pull/142034

I don't think the Kibana sample data logs index is a good one to use.
I'll find out if we can use the data I was using for testing